app/javascript/Pages/components/Navbar/navbar.scss

```diff
@@ -0,0 +1,186 @@
+@import "../../../../../node_modules/mappy-breakpoints/mappy-breakpoints.scss";
+
+$breakpoints: (
+  "small": 320px,
+  "large": 960px,
+);
+
+$primary-color: #6600cc;
+$dark-color: #252636;
+$white-color: #ffffff;
+$gray-color: #aaaaaa;
+
+.navbar {
+  background: $primary-color;
+  height: 5rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.2rem;
+  position: sticky;
+  top: 0;
+  z-index: 999;
+
+  .navbar-container {
+    display: flex;
+    justify-content: space-between;
+    height: 5rem;
+    z-index: 1;
+    width: 100%;
+    max-width: 81.25rem;
+    margin-right: auto;
+    margin-left: auto;
+    padding-left: 3.1rem;
+
+    .navbar-logo {
+      color: $white-color;
+      justify-self: start;
+      cursor: pointer;
+      margin-top: 2%;
+    }
+
+    .navbar-menu-icons {
+      color: $white-color;
+
+      .navbar-menu-icon-notification {
+        display: flex;
+        cursor: pointer;
+        position: absolute;
+        top: 50%;
+        transform: translate(0, -50%);
+        font-size: 1.8rem;
+        padding: 2rem;
+        right: 0;
+
+        &:hover {
+          font-size: 2rem;
+          transition: all 0.2s ease-out;
+        }
+      }
+      .navbar-menu-icon-burger {
+        display: none;
+      }
+    }
+
+    .nav-menu {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      list-style: none;
+      text-align: center;
+      justify-content: end;
+
+      .nav-item {
+        height: 5rem;
+        border-bottom: 2px solid transparent;
+
+        &:hover {
+          border-bottom: 4px solid $white-color;
+        }
+
+        .nav-links {
+          color: $white-color;
+          display: flex;
+          align-items: center;
+          text-decoration: none;
+          padding: 0.5rem 2rem;
+          height: 100%;
+        }
+      }
+
+      .logout {
+        padding: 0.5rem 2rem;
+
+        .btn-logout {
+          border: none;
+          cursor: pointer;
+          white-space: nowrap;
+          background-color: transparent;
+          font-size: 1rem;
+          color: $gray-color;
+        }
+      }
+    }
+  }
+}
+
+@include mappy-bp(small large) {
```

Comment:
**issue (blocking)**: this is where you should indent, instea of having the responsive styles separate from the others you can indent them like:
```
.navbar {
    // main styles
    @include mappy-bp(large) {
        // breakpoint styles
    }
}
```
this way all the styles for the same element are together and you don't need to repeat the class name twice.

Also, the best practice we like to follow is to style elements from the smallest breakpoint to the largest. So, the main styles (without media query) would be the mobile, then you add a breakpoint for each time the website changes (tablet, desktop), but they should all be "min-width" and don't have max (so that styles compound and override)

Reply:
Ok, I not did that in the previous elements, but here forward I will! (and if it's possible correct the old ones)